### PR TITLE
Add Polkadex parathread/parachain information

### PR DESCRIPTION
**Pull request by arrudagates**
This PR adds information for the ui to display Polkadex parathread data.

### CODE REVIEW

**Review by jacogr — CHANGES_REQUESTED** [review]
Thank you. 

Some CI linting issues, spaces vs tabs - https://github.com/polkadot-js/apps/runs/4842909982?check_suite_focus=true#step:3:689

I also believe the ordering may be incorrect here, generally would like the names in alphabetical order. (There is a test on CI that runs that will fail if not)

File: packages/apps-config/src/endpoints/productionRelayPolkadot.ts [hunk 1]
```diff
@@ -266,6 +266,15 @@ export function createPolkadot (t: TFunction): EndpointOption {
...
           SubGame: 'wss://gamma.subgame.org/'
         }
+      },
+      {
+        info: 'polkadex',
+        homepage: 'https://polkadex.trade/',
+        paraId: 2036,
+        text: t('rpc.polkadot.polkadex', 'Polkadex', { ns: 'apps-config' }),
+        providers: {
+        	'Polkadex Team': 'wss://mainnet.polkadex.trade/'
```
Comment (jacogr) [inline packages/apps-config/src/endpoints/productionRelayPolkadot.ts:276]:
```suggestion
          'Polkadex Team': 'wss://mainnet.polkadex.trade/'
```

**Review by jacogr — CHANGES_REQUESTED** [review]
After the linting fix applied above, CI is indeed failing on the alphabetical ordering - https://github.com/polkadot-js/apps/runs/4852151797?check_suite_focus=true#step:3:753

File: packages/apps-config/src/endpoints/productionRelayPolkadot.ts (hunk 1 above)
Comment (simonsso) [reply packages/apps-config/src/endpoints/productionRelayPolkadot.ts]: The URL is good

**Review by jacogr — APPROVED** [review]

**Pull request merged**